src/MSBuildLocator/build/TraverseRuntimeIdentifiers.targets

```diff
@@ -0,0 +1,31 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="TraverseRuntimeIdentifiers" AfterTargets="Restore">
+    <PropertyGroup>
+      <RuntimeIdentifiers>
+        linux-x64;
+        linux-musl-x64;
+        linux-arm;
+        linux-arm64;
+        linux-bionic-arm64;
+        linux-musl-arm64;
+        linux-musl-arm;
+        linux-bionic-x64;
+        osx-x64;
+        osx-arm64
+      </RuntimeIdentifiers>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <RuntimeIdentifierArray>$(RuntimeIdentifiers.Split(';'))</RuntimeIdentifierArray>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <CurrentRuntimeIdentifier Include="$(RuntimeIdentifierArray)" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="runtime.%(CurrentRuntimeIdentifier.Identity).Microsoft.NETCore.DotNetAppHost" Version="7.0.9" />
```

Comment:
I don't think I understand what this does. Can you elaborate, please?

Reply:
I try to resolve this comment https://github.com/microsoft/MSBuildLocator/pull/224#discussion_r1279191978
I list all possible platforms and try to copy what is avaliable locally to the output folder.

$(NETCoreSdkRuntimeIdentifier) usage I liked more.
Not sure if this the right way.